challenges/objects-arrays.js

```diff
@@ -124,22 +143,29 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
```

Comment:
I would try to refrain from unnecessary pushes, these array methods already create a new array.